tests/pass0/t03_basic_stack.test

```diff
@@ -0,0 +1,18 @@
+discard """
+  output: "(Done 4100)"
```

Comment:
Note to self: check why this address is stable

Reply:
All virtual address in the VM start at 4096 (accessing anything below that results in an access violation). The in-memory stack is located at the very start of the VM's memory region, and since the procedure is the very first procedure to run and allocate stack-space, its stack-pointer will be 4096. `(Addr (IntVal 4))` returns the stack-pointer + 4, hence 4100.

Relying on implementation details like this makes the test brittle, but only making sure the test compiles and runs wouldn't guarantee that stack allocation actually works. I think this tests needs to check the generated VM bytecode instead.